docs/api/symbol-package-publish-resource.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Push and Delete, NuGet API | Microsoft Docs
```

Comment:
The title is wrong. It should be `Push Symbol Packages, NuGet API | Microsoft Docs`

Reply:
Updated.